### PR TITLE
WIP: Eradicate facade feature from btcpay - it sucks and is useless 

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

**Comment by NicolasDorier** [comment]
tests not passing

File: BTCPayServer/Security/BitpayAuthentication.cs [hunk 1]
```diff
@@ -201,15 +201,7 @@ private async Task<BitTokenEntity> GetTokenPermissionAsync(string sin, string ex
 
             private IEnumerable<BitTokenEntity> GetCompatibleTokens(BitTokenEntity token)
```
Comment (NicolasDorier) [inline BTCPayServer/Security/BitpayAuthentication.cs]: maybe can remove this one

**Comment by Kukks** [comment]
still WIP, a lot of things i need to figure out on this considering there is a lot of schema stuff and bitpay compatibility things

**Comment by NicolasDorier** [comment]
yeah be careful, should not remove stuff in the JSONs though.

**Comment by NicolasDorier** [comment]
4067d4b00f0e2cffedffa06600f791d8efaed974

**Pull request closed without merging**